modules/react-components/src/components/links/link.tsx

```diff
@@ -62,21 +66,18 @@ interface LinkPropsInterface extends IdentifiableComponentInterface {
  * @param props - Props injected to the component.
  * @returns the link component.
  */
-export const Link: FunctionComponent<PropsWithChildren<LinkPropsInterface>> = (
-    props: PropsWithChildren<LinkPropsInterface>
-): ReactElement => {
-
-    const {
-        children,
-        className,
-        external,
-        icon,
-        iconPosition,
-        link,
-        onClick,
-        target,
-        [ "data-componentid" ]: componentId
-    } = props;
+export const Link: FunctionComponent<PropsWithChildren<LinkPropsInterface>> = ({
+    children,
+    className,
+    external = true,
```

Comment:
Any specific reason we moved the default props to the function declaration?

Reply:
The current pattern is to avoid using `defaultProps`. We should add those as normal props in a functional component.
Refer: https://github.com/facebook/react/pull/16210 (Confirmed by brion)